crates/zkvm/adapters/native/Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "strata-native-zkvm-adapter"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+bincode.workspace = true
+borsh.workspace = true
+serde.workspace = true
+strata-zkvm.workspace = true
+tracing.workspace = true
```

I prefer to order them as internal vs external:
```suggestion
strata-zkvm.workspace = true

bincode.workspace = true
borsh.workspace = true
serde.workspace = true
tracing.workspace = true
```